lib/broccoli/ember-app.js

```diff
@@ -1079,7 +1116,9 @@ class EmberApp {
         annotation: 'TreeMerger: `addon/` trees',
       });
 
-      this._cachedAddonTree = new Funnel(combinedAddonTree, {
+      let compiledAddonTree = this._compileAddonTree(combinedAddonTree);
```

Comment:
Can we add some broccoli-debug trees for before and after?  I'm mostly thinking this will be super useful for seeing where some transpilation issues _might_ be coming from...

Reply:
I emulated the style from the assembler. @twokul you may want to verify.